modules/Makefile.kubernetes

```diff
@@ -71,6 +72,15 @@ KUBECTL_CMD ?= $(KUBECTL_SSH_CMD) "$(KUBECTL)" --logtostderr=true --insecure-ski
 # Reference Docs:
 #  - https://cloud.google.com/container-engine/docs/kubectl/
 #
+## Bring up SSH tunnel for kubernetes commands, must be called before executing any targets
+kubernetes\:tunnel-up:
```

Comment:
You will need to ensure this gets called before all other targets. At the least this target: https://github.com/sagansystems/build-harness/blob/54cd63cebfaa666a55edf7bc206094b7fb2c9a11/modules/Makefile.kubernetes#L150

However, some of our builds may be calling sub-targets. Perhaps @osterman knows a way to auto apply this to all relevant targets